src/components/SideBar/index.tsx

```diff
@@ -1,14 +1,62 @@
 import React from 'react';
 
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+
 import './styles.scss';
+import { AppAction, SectionName } from '../../redux/actionTypes';
+import { AppState } from '../../redux/reducer';
+import { changeSection } from '../../redux/actions';
+import { DispatchProps } from '../dispatchProps';
```

Comment:
i think imports are organized by the filenames so the first one would be `import { DispatchProps } from '../dispatchProps';`